installation/install.sh

```
@@ -4507,6 +4508,14 @@ source ~/.bashrc >/dev/null 2>&1
 #Do preinstallation checking for CentOS and Ubuntu
 check_system
 
+if [ x"$PRECHECK" = x'y' ]; then
```

Shellcheck (0.11.0)

[style] Avoid x-prefix in comparisons as it no longer serves a purpose.

(SC2268)